ts/components/FiscalCodeComponent.tsx

```diff
@@ -36,12 +36,12 @@ interface PreviewProps {
 
 interface FullProps extends BaseProps {
   type: "Full";
-  getBackSide: boolean;
+  getBackSide?: boolean;
 }
 
 interface LandscapeProps extends BaseProps {
   type: "Landscape";
-  getBackSide: boolean;
+  getBackSide?: boolean;
 }
 
 type Props = PreviewProps | FullProps | LandscapeProps;
```

Comment:
```suggestion
type Props = PreviewProps | BaseProps;
```
this can be done by avoid code duplication
```typescript
interface BaseProps {
  profile: UserProfile;
  municipality: MunicipalityState;
  getBackSide: boolean;
  type: "Full" | "Landscape";
}
```
Indeed you could avoid optional for **getBackSide** since the passed value is always defined